packages/flutter_tools/lib/src/commands/build.dart

```diff
@@ -62,7 +60,8 @@ class BuildCommand extends FlutterCommand {
   String get category => FlutterCommandCategory.project;
 
   @override
-  Future<FlutterCommandResult> runCommand() async => FlutterCommandResult.fail();
+  Future<FlutterCommandResult> runCommand() async =>
```

Comment:
@christopherfujino Just confirming that the Flutter team is okay with auto-formatting all of these files? I wasn't sure if they weren't previously autoformatted to avoid messing with the `git blame`.

Reply:
ahh yeah, this particular line should be reverted to align with Flutter's style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

Good catch.

Reply:
the diff on lines 86-89 look fine to me

Reply:
done